packages/core/src/components/DateField/DateField.scss

```diff
@@ -33,6 +33,16 @@ Markup:
 
 Style guide: components.date-field
 */
+
+input[type="number"]::-webkit-outer-spin-button, input[type="number"]::-webkit-inner-spin-button {
```

Comment:
These selectors will apply to all text fields of the number type rather than just the date-field ones. Is that your intention?

Reply:
And I see that the ticket requests that the fields be turned into normal text fields. I like your solution better, but I just thought I'd bring that up.

Reply:
@pwolfert it wasn't, but thinking about it I think it's all right that all number fields don't have the increment arrows. They're a confusing UI and they don't always work as expected, which was the genesis for getting rid of them in the date field. 

RE: turning into normal text fields, discussed that with Mel and we agreed that retaining the number fields and removing the arrow UI was a good solution -- the problem is not the functionality of the number field per se, but of the increment arrows themselves. (the original AC is too prescriptive in this case)

Reply:
Cool, sounds good to me. In this case I'd suggest we move this into https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/TextField/TextField.scss#L45

Reply:
Yeah, that's the linter. It's rather opinionated 😕. I'll pull down your branch and try to fix them. You mind if I commit to your branch?

Also, none of our existing CSS has ever selected on the bare `input`s. I think it would be more in line with the existing pattern if the selector were `.ds-c-field[type="number"]...etc` that way we don't overstep our bounds.

Reply:
No, please go ahead! Appreciate the help.

Reply:
I'll try to get to it this morning, but if I don't make any updates in the next two hours, yell at me.